content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,61 @@ positional arguments:
+[parameters file](#parameters-file), where the _parameter value_ should be
+found. This allows you to define [stage](/doc/command-reference/run)
+dependencies more granularly: changes to other parts of the params file will not
+affect the stage. Parameters also prevent situations where several stages share
+a common dependency (e.g. a config file), and any change in it invalidates all
+these stages, causing unnecessary re-executions upon `dvc repro`.
+
+To start tracking parameters from the default `params.yaml` file, list them
+under the `params` field of `dvc.yaml` stages (manually or with the the
+`-p`/`--params` option of `dvc run`). Other params files can be listed too, with
+sub-lists of params names in them:
 
-The default parameters file name is `params.yaml`. Parameters should be
-organized as a tree hierarchy inside, as DVC will locate param names by their
-tree path. Parameters files have to be manually written, or generated, and these
-can be versioned directly with Git.
+```yaml
+stages:
+  mystage:
+    cmd: ./myscript.sh
+    params:
+      - foo # from params.yaml
+      - myparams.toml:
+          - bar
+          - baz.qux
+```
 
-Supported parameter _value_ types are: string, integer, float, and arrays. DVC
-itself does not ascribe any specific meaning for these values. They are
-user-defined, and serve as a way to generalize and parametrize an machine
-learning algorithms or data processing code.
+The `dvc params diff` command is available to show parameter changes, displaying
+their current and previous [values](#parameter-values).
 
-DVC saves the param names and their latest values in the `dvc.yaml` file. These
-values will be compared to the ones in the params files to determine if the
-stage is invalidated upon pipeline [reproduction](/doc/command-reference/repro).
+### Parameters files
 
-> Note that DVC does not pass the parameter values to stage commands. The
-> associated command executed by `dvc run` or `dvc repro` will have to open and
-> parse the parameters file by itself, and use the params specified with `-p`.
+The default params file name is `params.yaml`, but any other YAML 1.2, JSON,
+TOML, or [Python](#examples-python-parameters-file) files can be used
+additionally. These files are typically written manually (or they can be
+generated) and they can be versioned directly with Git.
 
-The parameters concept helps to define [stage](/doc/command-reference/run)
-dependencies more granularly. A particular parameter or set of parameters will
-be required for the stage invalidation (see `dvc status` and `dvc repro`).
-Changes to other parts of the dependency file will not affect the stage. This
-prevents situations where several stages share a (configuration) file as a
-common dependency, and any change in this dependency invalidates all these
-stages and causes their reproduction unnecessarily.
+Param values should be organized in tree-like hierarchies (dictionaries) inside
```

Comment:
you have `Param values` here and then immediately `### Parameter values` as a title - looks strange a bit

Reply:
Moved this into the Params values section for now.